core/src/main/scala/scalan/Base.scala

```diff
@@ -467,7 +467,7 @@ abstract class Base { scalan: Scalan =>
       * and leave non-Ref items unchanged. */
     final def apply(xs: Seq[Any])(implicit o: Overloaded1): Seq[Any] = {
       val len = xs.length
-      if (len == 0) mutable.WrappedArray.empty
+      if (len == 0) Seq[Any]()
```

Comment:
mutable.WrappedArray should be replaces with scala.collection.compat.immutable.ArraySeq. 
See recent commits in #807 where I replaces some of them.

Also, keep in mind that Sigma is almost entirely is part of hotspot code. 
Thus performance is critical and every allocation counts (basically every line is executed millions of times).
While migrating, introducing new allocations is not acceptable.
Please read https://github.com/ScorexFoundation/sigmastate-interpreter/blob/develop/docs/perf-style-guide.md for details of why Seq() is bad, and also for other performance patterns used in Sigma.

Reply:
Should i change all occurence of Array to ArraySeq?